server/migrations/20170328005425-create-game.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
```

Comment:
Actually that's a generated migration from sequelize, I didn't write any of that.